src/constants.py

```diff
@@ -1 +1,46 @@
+from pathlib import Path
+import pandas as pd
+
 COLOR_SCHEME = ["#003f5c", "#ffa600", "#58508d", "#ff6361", "#bc5090"]
+
+SQL_FILE_DIRECTORY = Path().absolute() / ".data/sql"
+
+DATASET_BY_VERSION = lambda dataset, version: f"{dataset}_{version.replace('/', '_')}"
```

Comment:
Nitpick: I don't like a function being all uppercase, even if it serves a similar role to the constants/literals in this file

Reply:
good call, will rename!

do you feel like `SQL_FILE_DIRECTORY` is enough of a constants to stay uppercase? since it doesn't have any inputs and is used as a constant

Reply:
Yeah that seems fine